sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -674,6 +674,27 @@ public void flush(long timeoutMillis) {
     return traceHeader;
   }
 
+  @Override
+  public @Nullable TraceStateHeader traceStateHeader() {
```

Comment:
We don't need it on `Sentry`, but we do need it on `IHub` so that integrations - like OkHttp one, could set the header. At least thats one way to do it and I don't know yet about another way.